src/gui/mzroll/database.h

```diff
@@ -74,11 +102,12 @@ class Database {
 	deque<Molecule2D*> coordinatesDB;
 
 	map<string, Compound*> compoundIdMap;
-	map<string, Reaction*> reactionIdMap;
+        map<string, Reaction*> reactionIdMap;
 	map<string, Pathway*> pathwayIdMap;
 	map<string, Molecule2D*> coordinatesMap;
     vector<string> notFoundColumns;
     vector<string> invalidRows;
+    map<string, int> prmIdCount;
```

Comment:
idCount is also applicable in case of MRM compound lists where different fragments of a precursor will have the same ID. Would suggest changing the name to something more generic. `idCount` works.